internal/util/projutil/project_util.go

```diff
@@ -323,3 +324,23 @@ func PrintDeprecationWarning(msg string) {
 	fmt.Printf(noticeColor, "[Deprecation Notice] "+msg+". Refer to the version upgrade guide "+
 		"for more information: https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/\n\n")
 }
+
+// RewriteFileContents adds the provided content before the last occurance of the word label
+// and rewrites the file with the new content.
+func RewriteFileContents(filename, label, content string) error {
+	text, err := ioutil.ReadFile(filename)
+
+	if err != nil {
+		return fmt.Errorf("error in getting contents from the file, %v", err)
+	}
+
+	existingContent := string(text)
+	index := strings.LastIndex(existingContent, label)
```

Comment:
To append after the last instance of `label`, do something like:
```suggestion
	labelIndex := strings.LastIndex(existingContent, label)
	index := strings.Index(existingContent[labelIndex:], "\n") + 1
```
And to be safe, error if indices are `-1`.